src/test/ui/privacy/where-priv-type.rs

```diff
@@ -0,0 +1,83 @@
+#![crate_type = "lib"]
```

Comment:
Could you add a comment explaining what this test is testing in a couple of sentences?

Reply:
(And to the second new test too.)

Reply:
Resolved by https://github.com/rust-lang/rust/pull/90586/commits/932a34ce2005ccfae225aab2bf424e7233d54e48.